hub/policy/src/test/java/uk/gov/ida/hub/policy/services/AuthnResponseFromIdpServiceTest.java

```diff
@@ -96,11 +97,29 @@ public void shouldSendRequestToMatchingServiceViaAttributeQueryServiceAndUpdateS
         // Then
         verify(samlAuthnResponseTranslatorDtoFactory).fromSamlAuthnResponseContainerDto(samlAuthnResponseContainerDto, msaEntityId);
         verify(attributeQueryService).sendAttributeQueryRequest(sessionId, attributeQueryRequestDto);
-        verifyIdpStateControllerIsCalledWithRightDataOnSuccess(successResponseFromIdp);
+        verifyIdpStateControllerIsCalledWithRightDataOnSuccess(successResponseFromIdp, true);
         ResponseAction expectedResponseAction = ResponseAction.success(sessionId, REGISTERING, loaAchieved);
         assertThat(responseAction).isEqualToComparingFieldByField(expectedResponseAction);
     }
 
+    @Test
+    public void shouldReturnNonMatchingJourneySuccessWhenSuccessfulResponseIsReceivedForAServiceWithNoMatching() {
```

Comment:
Just to be more precise `shouldReturnNonMatchingJourneySuccessWhenSuccessfulResponseIsReceivedForAServiceWithN**oMatching**`  is better named as `shouldReturnNonMatchingJourneySuccessWhenSuccessfulResponseIsReceivedForAServiceWithNonMatching`
Since there is a NoMatching concept for Matching Journey when a match is not found

Reply:
Well spotted - yep, that was a typo and I meant nonMatching.  I'll push a fix.